### PR TITLE
Show all output files, even nested ones

### PR DESCRIPTION
**Pull request by cdebourcy**

### CODE REVIEW

**Comment by cdebourcy** [comment]
Yes, just added that

File: app/models/sample.rb [hunk 1]
```diff
@@ -105,11 +105,25 @@ def self.get_signed_url(s3key)
     nil
   end
 
-  def results_folder_files
+  def path_end(key, n = 1)
+    parts = key.split('/')
+    return "#{parts[-2]}/#{parts[-1]}" if n == 2
```
Comment (yunfangjuan) [inline app/models/sample.rb]: n == 2 ? "#{parts[-2]}/#{parts[-1]}" : parts[-1]

File: app/models/sample.rb [hunk 2]
```diff
@@ -105,11 +105,25 @@ def self.get_signed_url(s3key)
...
+  end
+
+  def results_folder_files
+    prs = pipeline_runs.first.pipeline_run_stages.first
+    stage1_files = list_outputs(prs.sample_output_s3_path)
+    stage2_files = list_outputs(prs.alignment_output_s3_path, 2)
+    stage1_files + stage2_files
   end
 
   def fastqs_folder_files
```
Comment (yunfangjuan) [inline app/models/sample.rb]: for consistency reasons, you should change the format here too.

File: app/models/sample.rb [hunk 3]
```diff
@@ -105,11 +105,25 @@ def self.get_signed_url(s3key)
     nil
   end
 
-  def results_folder_files
+  def path_end(key, n = 1)
```
Comment (yunfangjuan) [inline app/models/sample.rb]: I think end_path might be a better name.

File: app/models/sample.rb (hunk 2 above)
Comment (cdebourcy) [reply app/models/sample.rb]: ok!

**Pull request merged**